### PR TITLE
Multilabel Brazilian Toxic Tweets Classification

### PR DESCRIPTION
**Pull request by dokato**
## Checklist for adding MMTEB dataset

Reason for dataset addition: there's shortage of datasets with Brazilian Portugise, and currently we don't have big enough variety of multilabel datasets. This one is well structured and described: https://paperswithcode.com/dataset/told-br 

Furthermore, I corrected some minor issues with Maltese News Classification.

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [x] I have added points for my submission to the [points folder](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) using the PR number as the filename (e.g. `438.jsonl`).

### CODE REVIEW

**Comment by dokato** [comment]
@x-tabdeveloping sorry to bother again, but I need advise how to handle this set. On [HF](https://huggingface.co/datasets/told-br/viewer/multilabel) it consists of 1 train split with 21k examples.
Here again I can't use built-in stratified `train_test_split` from `datasets` as it complains about column type, so I just use random samples. But as a consequence, we can't guarantee that the labels from training will match ones in test. WDYT? Should we sort https://github.com/embeddings-benchmark/mteb/pull/760 or https://github.com/embeddings-benchmark/mteb/pull/694 first?

**Comment by x-tabdeveloping** [comment]
As far as I know we stopped dataset submissions on the 15th.
I think we should focus our efforts on speeding up the benchmark and finalising everything before running the models.

**Comment by x-tabdeveloping** [comment]
I see you have some changes in Maltese news. Can you elaborate on what you did, and why?

**Comment by x-tabdeveloping** [comment]
@Ruqyai The checklist is incomplete and the name of the PR is set to "work in progress". I believe it would be a tad irresponsible to merge this, no? Seems a bit too early and undercooked to just LGTM it to me.

**Comment by x-tabdeveloping** [comment]
I see your point! Can I ask you to move the changes related to the Maltese News to another PR so we can discuss them separately? (seems quite reasonable otherwise)
@KennethEnevoldsen what is your take on this? Should we consider adding this still or stick to the deadline we set to ourselves?

File: mteb/tasks/MultiLabelClassification/por/BrazilianToxicTweetsClassification.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            journal   = {CoRR},
+            volume    = {abs/2010.04543},
+            year      = {2020},
+            url       = {https://arxiv.org/abs/2010.04543},
+            eprinttype = {arXiv},
+            eprint    = {2010.04543},
+            timestamp = {Tue, 15 Dec 2020 16:10:16 +0100},
+            }""",
+        n_samples={"train": 2048, "test": 2048},
+        avg_character_length={"train": 88.45, "test": 85.05},
```
Comment (KennethEnevoldsen) [inline mteb/tasks/MultiLabelClassification/por/BrazilianToxicTweetsClassification.py:52]:
```suggestion
        n_samples={"test": 2048},
        avg_character_length={"test": 85.05},
```

**Review by KennethEnevoldsen — APPROVED** [review]
> @KennethEnevoldsen what is your take on this? Should we consider adding this still or stick to the deadline we set to ourselves?

This looks fine to merge in for me - @dokato will you fill out the checklist
